.github/workflows/preview.yaml

```
@@ -53,11 +56,8 @@ jobs:
       - name: Generate preview domain names
         id: domains
         run: |
-          REF="${{ github.head_ref }}"
-          CLEAN_REF="${REF//[^a-z0-9\-]/x}"
-
-          PREVIEW_ALIAS="${CLEAN_REF}.preview.avy-fx.org"
-          WILDCARD_PREVIEW_ALIAS="*.${CLEAN_REF}.preview.avy-fx.org"
+          PREVIEW_ALIAS="${{ steps.database.outputs.name }}.preview.avy-fx.org"
```

```suggestion
          PREVIEW_ALIAS="${{ steps.database.outputs.ref }}.preview.avy-fx.org"
```

Since the `name` is the `payloadcms-preview-${ref}`